users/models/identity.py

```diff
@@ -534,7 +534,7 @@ def to_ap(self):
         if self.metadata:
             response["attachment"] = [
                 {
-                    "type": "http://schema.org#PropertyValue",
```

Comment:
Took some time to understand that Mastodon sends and expects to receive `PropertyValue` instead of `http://schema.org#PropertyValue` because the original payload is hidden behind the call to `canonicalize` that expands the original value.